packages/catalog-process/src/services/catalogService.ts

```diff
@@ -84,17 +87,6 @@ const assertRequesterAllowed = (
   }
 };
 
-export const retrieveEService = async (
-  eserviceId: EServiceId,
-  readModelService: ReadModelService
-): Promise<WithMetadata<EService>> => {
-  const eService = await readModelService.getEServiceById(eserviceId);
-  if (eService === undefined) {
-    throw eServiceNotFound(eserviceId);
-  }
-  return eService;
-};
-
```

Comment:
Could we keep this function? I know it's currently unused, but I just started using it in other pull requests.